src/Sulu/Bundle/AdminBundle/Resources/js/containers/Toolbar/types.js

```diff
@@ -49,5 +49,5 @@ export type ToolbarConfig = {
 
 export interface ToolbarAction {
```

Comment:
not sure if this interface is still in the right place. it is used only in the `AbstractListToolbarAction` and the `AbstractFormToolbarAction` but nowhere inside the `Toolbar` directory

Reply:
I think it still makes sense, because it makes sure that the toolbar actions for forms and list implement the same methods. Otherwise the two implementation might diverge.